soh/src/code/z_play.c

```diff
@@ -1087,6 +1087,14 @@ void Play_Update(PlayState* play) {
                 // Gameplay stat tracking
                 if (!gSaveContext.sohStats.gameComplete) {
                       gSaveContext.sohStats.playTimer++;
+                      // Use ITEM_KEY_BOSS for Ganon's boss key
+                      if ((gSaveContext.inventory.dungeonItems[10] & 1) &&
+                          gSaveContext.sohStats.timestamp[ITEM_KEY_BOSS] == 0) {
+                          gSaveContext.sohStats.timestamp[ITEM_KEY_BOSS] = GAMEPLAYSTAT_TOTAL_TIME;
+                      }
+                      if (CVar_GetS32("gMMBunnyHood", 0) && Player_GetMask(play) == PLAYER_MASK_BUNNY) {
+                          gSaveContext.sohStats.count[COUNT_TIME_BUNNY_HOOD]++;
+                      }
```

Comment:
i feel like it'd be nice to have this logic somewhere else, but i'm not sure where else to put it

i think i'm mostly just a fan of how simple this was before when it was
```c
if (!gSaveContext.sohStats.gameComplete) {
    gSaveContext.sohStats.playTimer++;
}
```

Reply:
Maybe this can be pulled out into its own function and placed above? Something like `GameplayStats_UpdateGameplayTimers()`, which can then hold the bunny hood timer and any other future timers if people want them.  (One thought I had was maybe keeping track of how much time the player has spent in each dungeon, and logic for this could live in such a function.)